api/opencl_runtime_layer.asciidoc

```diff
@@ -15110,7 +15158,7 @@ ifdef::cl_khr_command_buffer_mutable_dispatch[]
     The {cl_khr_command_buffer_EXT} extension does not define any
     properties, but supported properties defined by extensions are defined
     in the <<ndrange-kernel-properties-table, List of supported properties
-    by {clCommandNDRangeKernelKHR}>> table.
+    by clCommandNDRangeKernelKHR>> table.
```

Comment:
Curious: did use of the asciidoctor attribute here cause problems?

This is fine for now, but at some point we might want to rename this table because using "clCommandNDRangeKernelKHR" without the asciidoctor attribute will get flagged by some of the spec checkers.

Reply:
Interesting, in my local build I'm seeing this
![image](https://github.com/user-attachments/assets/f66aff3d-701c-43aa-b927-206e15e9d361)
But that doesn't seem to be a problem in the hosted render. Wonder if this is another data point for https://github.com/KhronosGroup/OpenCL-Docs/issues/1175 on inconsistent rendering 

Happy to remove to keep this change as preferred.